docs/DesignDocs/InitTimeConfigurationInterface.md

```diff
@@ -83,28 +83,66 @@ typedef enum _oe_enclave_setting_type
      *  reflected in TEE-produced enclave identity evidence. Currently only
      *  supported by SGX Enclaves with KSS feature enabled.
      */
-    OE_SGX_CONFIGURATION_DATA = 0x78b5b41d
+    OE_SGX_ENCLAVE_CONFIG_DATA = 0x78b5b41d
 } oe_enclave_setting_type_t;
 
-typedef struct _oe_sgx_configuration_data
+typedef struct _oe_sgx_enclave_setting_config_data
```

Comment:
avoid "config" abbreviation

Reply:
As this is already defined in public header file in include/openenclave/host.h file and was part of previous release, it would create compatibility issues if we change it now.

Reply:
Let us not change what is in the public headers.